src/test/ui/array-slice-vec/suggest-array-length.stderr

```diff
@@ -0,0 +1,48 @@
+error: in expressions, `_` can only be used on the left-hand side of an assignment
```

Comment:
Why are all errors in this file duplicated? For example this is and the one at line 28 has the same cause.